### ISSUE TITLE
[Feature Request]: Work with a document

### ISSUE THREAD

**Issue by ralienpp** [infiniflow/ragflow#1424]
### Is there an existing issue for the same feature request?

- [X] I have checked the existing issues.

### Describe the feature you'd like

This can be seen as an ad-hoc knowledge-base + chat bot duo. The use-case is as follows:

- I have a new document that I need to familiarize myself with, it could be hundreds of pages long
- I'd like to press a button called "Work with a document" which allows me to upload it into the system
- In the background, a new knowledge-base and bot created with default settings (but there is a tab where I can customize it to my needs)
- I can then ask questions about this specific document, as usual

The reason I think this could be a separate option is because many of these documents will be of the "read once and throw it away" type; I don't want to create a new knowledge-base for something like that. Perhaps they could be stored in a temporary knowledge base, which I can then choose to merge into an existing knowledge-base if the document is useful; otherwise the temporary knowledge base will be deleted when the tab is closed or after a period of time.

In other words, this is a high-level wrapper on top of the existing functionality, which accommodates working with "throw-away" materials.

**Comment by JinHai-CN** [comment]
In new version, the document can be uploaded in chat window and chat with it. But this temporary knowledge base can't be 'promoted' to merge with existing document, right know.

**Comment by which-W** [comment]
@ralienpp 
Thanks so much for your suggestion — and apologies for the late reply! ⏳

We’ve added support for temporarily uploading files 📎. Could you please confirm if this meets your needs? If so, feel free to close this feature request — otherwise, we’ll include it in our next cleanup pass ✅.

Always happy to hear more constructive ideas to help improve the project — keep them coming! 💬🚀

Thanks again! 🙌
